lighthouse/helpers/plates.py

```diff
@@ -199,8 +204,68 @@ def row_to_dict(row):
     return obj
 
 
+def request_with_retries(request_func: Callable, response_func: Callable[[Any], T], max_retries=3) -> T:
+    attempt = 1
+
+    while attempt <= max_retries:
+        LOGGER.debug(f"Starting attempt {attempt} of {max_retries}...")
+
+        try:
+            response = request_func()
+
+            if response.status_code == 200:
+                return response_func(response)
+
+            LOGGER.debug(
+                f"Attempt failed due to an invalid status code {response.status_code}. "
+                "Pausing 1 second before trying again."
+            )
+            attempt += 1
+            sleep(1)
+        except requests.ConnectionError:
+            LOGGER.debug("Attempt failed due to a connection error. Pausing 1 second before trying again.")
+            attempt += 1
+            sleep(1)
+
+    raise requests.ConnectionError("Unable to access Sequencescape.")
+
+
+def check_if_plate_exists_in_ss(uuid: str) -> bool:
+    ss_url = f"{app.config['SS_URL']}/api/v2/plates"
```

Comment:
I noticed that there are unit tests for other parts of this file that are using the `api/v2/labware` endpoint.  I could do the same, but I decided not to change it since I am specifically looking for plates.  Barcodes are shared between types of labware though, so if you think it would be better, this could be changed to `labware` and the unit tests updated accordingly.